membership-attribute-service/app/controllers/AttributeController.scala

```diff
@@ -175,10 +167,10 @@ class AttributeController(
         }).recover { case e =>
           // This branch indicates a serious error to be investigated ASAP, because it likely means we could not
           // serve from either Zuora or DynamoDB cache. Likely multi-system outage in progress or logic error.
-          val errMsg = s"Failed to serve entitlements either from cache or directly. Urgently notify Retention team: $e"
+          val errMsg = scrub"Failed to serve entitlements either from cache or directly. Urgently notify Retention team: $e"
           metrics.incrementCount(s"$endpointDescription-failed-to-serve-entitlements")
-          log.error(errMsg, e)
-          InternalServerError(errMsg)
+          logger.error(errMsg, e)
+          InternalServerError("failed to serve entitlements")
```

Comment:
shouldn't pipe actual exceptions into an InternalServerError response as it's a security bad practice